text/game/gui.stringtable

```diff
@@ -3671,12 +3671,12 @@ Beschwörungen - Mächtige magische Effekte, die Sänger auslösen können, wenn
     </Entry>
     <Entry>
       <ID>723</ID>
-      <DefaultText>In Aedyranisch als „Berath“ und in Vailianisch als „Cirono“ bekannt; ist der Gott der Kreise, der Türen und des Lebens und des Todes selbst. Gewöhnlich schnitzen oder platzieren Menschen die Figur von Berath in Türeingänge, Fenster und andere „Portale“, die von einem Ort zum anderen führen. Berath hat eine relativ kleine Priesterschaft. Zum Teil, weil er nicht oft zu ihnen spricht. Dennoch hat Berath viele Bittsteller und gelegentliche Anhänger.</DefaultText>
+      <DefaultText>In Aedyrisch als „Berath“ und in Vailianisch als „Cirono“ bekannt; ist der Gott der Kreise, der Türen und des Lebens und des Todes selbst. Gewöhnlich schnitzen oder platzieren Menschen die Figur von Berath in Türeingänge, Fenster und andere „Portale“, die von einem Ort zum anderen führen. Berath hat eine relativ kleine Priesterschaft. Zum Teil, weil er nicht oft zu ihnen spricht. Dennoch hat Berath viele Bittsteller und gelegentliche Anhänger.</DefaultText>
```

Comment:
Denke ja: Siehe [Punkt 2a](http://www.duden.de/rechtschreibung/deutsch): _Großgeschrieben wird das substantivierte Adjektiv, wenn es im Sinne von »deutsche Sprache« verwendet wird_

Die Anführungszeichen waren scheinbar schon in phobs [Initial Commit](https://github.com/phob/pillarsofeternity-german-patch/blob/ce787198509cfa8779f6b383d6fe6a96cdcbb2d3/text/game/gui.stringtable)